<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndLoadPointPicture</class>
 <widget class="QDialog" name="WndLoadPointPicture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0,4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="WndIrImage" name="id_alarm_widget_ir" native="true"/>
   </item>
   <item>
    <widget class="WndPalette" name="id_alarm_widget_palette" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="WndViImage" name="id_alarm_widget_vi" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WndPalette</class>
   <extends>QWidget</extends>
   <header>wndpalette.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WndIrImage</class>
   <extends>QWidget</extends>
   <header>wndirimage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WndViImage</class>
   <extends>QWidget</extends>
   <header>wndviimage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
